src/what-spaced-repetition-algorithm.md

```diff
@@ -108,15 +108,6 @@ increased beyond the value of *maximum interval*. Finally, all new
 intervals (except Again) will always be at least one day longer than the
 previous interval.
 
-After you select an ease button, Anki also applies a small amount of
-random “fuzz” to prevent cards that were introduced at the same time and
-given the same ratings from sticking together and always coming up for
-review on the same day. This fuzz appear on the interval
-buttons from 2.1.45+, but not in previous versions, so if you are using 
-a previous version and you’re 
-noticing a slight discrepancy between what you
-select and the intervals your cards actually get, this is probably the
-cause.
```

Comment:
This applies to the v1/v2 scheduler, not a specific Anki version. Given that not everyone is using the v3 scheduler yet, perhaps it is worth keeping in?

Reply:
Ah, just saw you moved this into the manual. Would it be worth a link at least?

Reply:
To me, this section makes more sense in the main documentation, if we move it there, I don't really see any reason why we should add a link to that section here. 
Of course, that's just me :)